src/library/components/table/types.ts

```diff
@@ -198,6 +199,13 @@ interface BaseColumnProps<T> {
   canEdit?: (row: T, column: DataColumn<T>) => boolean;
   preMDRColumnWidth?: number
   EditorWrapper?: React.ElementType<EditorWrapperProps<T>>
+  columnSearch?: ColumnSearch
+}
+
+type ColumnSearch = {
+  enabled: boolean
+  op?: AllFilterOperators
+  renderSearchButton?: boolean
```

Comment:
Hmm...

because the property on the column is optional - the existence of the setting should be enough that I don't think we need to manually specify the "enabled" prop below here

I also don't think we need to specify the `renderSearchButton` for _each_ column

Perhaps we just standardize on an onBlur triggers the change in filter and make the column setting columnSearchOperator

And unless you specify both the table setting AND this setting the column filter shows up or not.

```suggestion
  columnSearchOperator? AllFilterOperators
```